src/components/Tables/components/TableActionMenu.tsx

```diff
@@ -0,0 +1,66 @@
+import { DropdownMenu } from '@radix-ui/themes'
+import { DotsVerticalIcon } from '@radix-ui/react-icons'
+import { useState } from 'react'
+import { ConfirmDelete } from './ConfirmDelete'
+
+type Props = {
+    isFeatured: boolean
+    onToggleFeatured: () => void
+    onEdit: () => void
+    onDelete: () => void
+}
+
+export function TableActionMenu({
+    isFeatured,
+    onToggleFeatured,
+    onEdit,
+    onDelete
+}: Props) {
+    const [deleteDialogOpen, setDeleteDialogOpen] = useState(false)
+
+    return (
+        <>
+            <DropdownMenu.Root>
+                <DropdownMenu.Trigger>
+                    <DotsVerticalIcon style={{ cursor: 'pointer' }} />
+                </DropdownMenu.Trigger>
+                <DropdownMenu.Content size="2">
+                    <DropdownMenu.Item
+                        style={{ cursor: 'pointer' }}
```

Comment:
Almost! Please add the cursor via tailwind styles.

Reply:
ok this was my first thought but I wasn't sure if you wanted globals.scss to be modified or not.